script.js

```diff
@@ -0,0 +1,69 @@
+let developers = [
+    {
+        "name": "Carolin Bloeme",
+        "id": "carolin-bloeme"
+    },
+    {
+        "name": "Sun Jun",
+        "id": "sun-jun"
+    },
+    {
+        "name": "Song Bao",
+        "id": "song-bao"
+    },
+    {
+        "name": "Olivia Arribas",
+        "id": "olivia-arribas"
+    },
+    {
+        "name": "Samah abu laymun ",
+        "id": "samah-abu-laymun"
+    }
+]
+
+let ul = document.getElementById('developers');
+let li = "";
+for (let i = 0; i < developers.length; i++) {
+    li +=
+        `
+  <div class="list-item-wrapper">
+  <li
+  class="list-group-item d-flex justify-content-between align-items-center"
+>
+  <div class="item-name" id="${developers[i].id}-item">
+    <i class="fa fa-user m-1"></i> ${developers[i].name}
+  </div>
+
+  <label class="custom-checkbox">
+    <input
+      type="checkbox"
+      name="developer-names"
+      value="${developers[i].name}"
+      id="${developers[i].id}"
+    />
+    <span></span>
+  </label>
+  
+</li>
+  </div>
+  `;
+}
+document.getElementById('developers').innerHTML = li;
+var checkboxInputs = document.querySelectorAll("input[type='checkbox']");
+for (let i = 0; i < checkboxInputs.length; i++) {
+    checkboxInputs[i].addEventListener('click', function () {
+        if (checkboxInputs[i].checked) {
+            document.getElementById(checkboxInputs[i].id + "-item").style.color = "#1c1c1d";
```

Comment:
Is this done intentionally here and not in CSS (To let me know)? as I prefer doing it in CSS